cuqi/distribution/_beta.py

```diff
@@ -9,23 +9,17 @@ class Beta(Distribution):
     """
     Multivariate beta distribution of independent random variables x_i. Each is distributed according to the PDF function
 
-    f(x) = x^(alpha-1) * (1-x)^(beta-1) * Gamma(alpha+beta) / (Gamma(alpha)*Gamma(beta))
+    .. math::
 
-    where Gamma is the Gamma function.
+        f(x) = x^{(\\alpha-1)}(1-x)^{(\\beta-1)}\Gamma(\\alpha+\\beta) / (\Gamma(\\alpha)\Gamma(\\beta))
+
+    where :math:`\Gamma` is the Gamma function.
```

Comment:
it is nice to have the formula updated with symbols. I suggest to link the class parameters with the equation to be more explicit (may be something like this)
```
    Parameters
    ------------
    alpha: float or array_like
        The shape parameter :math:`\\alpha` of the beta distribution

    beta: float or array_like
        The shape parameter :math:`\\beta` of the beta distribution
```

Reply:
Done as suggested